src/libraries/Microsoft.Extensions.Logging.Console/src/AnsiLogConsole.cs

```diff
@@ -7,119 +7,20 @@
 namespace Microsoft.Extensions.Logging.Console
 {
     /// <summary>
-    /// For non-Windows platform consoles which understand the ANSI escape code sequences to represent color
+    /// For consoles which understand the ANSI escape code sequences to represent color
     /// </summary>
     internal class AnsiLogConsole : IConsole
     {
-        private readonly StringBuilder _outputBuilder;
         private readonly IAnsiSystemConsole _systemConsole;
 
-        public AnsiLogConsole(IAnsiSystemConsole systemConsole)
+        public AnsiLogConsole(bool stdErr = false)
         {
-            _outputBuilder = new StringBuilder();
-            _systemConsole = systemConsole;
+            _systemConsole = new AnsiSystemConsole(stdErr);
```

Comment:
We should just delete the `AnsiSystemConsole` class and `IAnsiSystemConsole` interface. They provide no value. That is what I was trying to say in my other comment here.